kythe/java/com/google/devtools/kythe/analyzers/java/KytheDocTreeScanner.java

```diff
@@ -86,10 +99,9 @@ public int transform(int pos) {
         JAVADOC,
         bracketed,
         anchoredTo,
-        deprecated,
         node,
         absNode == null ? null : absNode.getVName());
-    return true;
+    return DocCommentVisitResult.create(/* documented= */ true, /* deprecated= */ deprecated);
```

Comment:
Can the "deprecated" field just be a local now?

Reply:
Not without additional refactoring, the field is set in `KytheDocTreeScanner#visitDeprecated`.